app/javascript/admin/controllers/creator_settings_controller.js

```diff
@@ -70,13 +71,30 @@ export class CreatorSettingsController extends Controller {
   }
 
   /**
-   * Updates ths branding/colors on the Creator Settings Page.
-   *
+   * Validates the color contrast for accessibility,
+   * if the contrast is okay, it updates the branding,
+   * else it displays the error.
    * @param {Event} event
    */
-  updateBranding(event) {
+  handleValidationsAndUpdates(event) {
     const { value: color } = event.target;
 
+    if (isLowContrast(color)) {
+      this.colorContrastErrorTarget.innerText =
+        'The selected color must be darker for accessibility purposes.';
```

Comment:
Should we say "higher contrast" instead of "darker"?  I'm thinking if someone had Dark mode enabled.

And perhaps a hint as to what it should be higher contrast / darker than?

Reply:
@jeremyf when a creator first sets up their forem it will always be set up initially as a light theme, and as the feature stands once the creator completes their setup/onboarding, they will not be able to navigate back to their onboarding but they then go to the `/admin` to make changes. 

Hence I think, it's okay to leave it as is, since the language is in this case is unambiguous, they would just need to choose a darker color which when compared to white will always create a higher contrast.